### PR TITLE
[zh] synchronization kubespray.md

### PR DESCRIPTION
**Pull request by shuaijinchao**
/language zh

### CODE REVIEW

File: content/zh/docs/setup/production-environment/tools/kubespray.md [hunk 1]
```diff
@@ -191,16 +190,16 @@ Large deployments (100+ nodes) may require [specific adjustments](https://github
...
 -->
 ### （5/5）验证部署
 
 Kubespray 提供了一种使用
 [Netchecker](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/netcheck.md)
 验证 Pod 间连接和 DNS 解析的方法。
-Netchecker 确保 netchecker-agents pod 可以解析。
-DNS 请求并在默认名称空间内对每个请求执行 ping 操作。
-这些 Pods 模仿其余工作负载的类似行为，并用作集群运行状况指示器。
+Netchecker 确保 netchecker-agents Pods 可以解析DNS请求，
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubespray.md:200]: 英文前后加空格

Comment (shuaijinchao) [reply content/zh/docs/setup/production-environment/tools/kubespray.md]: done

**Comment by mengjiao-liu** [comment]
/lgtm
Thanks a lot for contributions！

**Pull request merged**